OpenOversight/app/models/database.py

```diff
@@ -312,13 +312,10 @@ def currently_on_force(self):
     def __repr__(self):
         if self.unique_internal_identifier:
             return (
-                f"<Officer ID {self.id}: {self.first_name} {self.middle_initial} "
-                + f"{self.last_name} {self.suffix} ({self.unique_internal_identifier})>"
+                f"<Officer ID {self.id}: {self.full_name()} "
+                f"({self.unique_internal_identifier})>"
             )
-        return (
-            f"<Officer ID {self.id}: {self.first_name} {self.middle_initial} "
-            + f"{self.last_name} {self.suffix}>"
-        )
+        return f"<Officer ID {self.id}: {self.full_name()}>"
```

Comment:
These changes will be masked by: https://github.com/lucyparsons/OpenOversight/pull/1037